shared/src/commonMain/kotlin/org/hyperskill/app/track_list/view/TrackListViewState.kt

```diff
@@ -0,0 +1,26 @@
+package org.hyperskill.app.track_list.view
+
+sealed interface TrackListViewState {
+    object Idle : TrackListViewState
+
+    object Loading : TrackListViewState
+
+    object Error : TrackListViewState
+
+    data class Content(
+        val selectedTrack: Track?,
+        val tracks: List<Track>
+    ) : TrackListViewState
+
+    @kotlinx.serialization.Serializable
```

Comment:
It was necessity because on android we passed this value to bottom sheet. In current implementation we don't have bottom sheet, so yes, I'll remove it